spring-cloud-dataflow-classic-docs/src/test/java/org/springframework/cloud/dataflow/server/rest/documentation/AuditRecordsDocumentation.java

```diff
@@ -84,10 +84,10 @@ public void listAllAuditRecords() throws Exception {
 					parameterWithName("actions")
 						.description("Comma-separated list of Audit Actions (optional)")),
 				responseFields(
-				fieldWithPath("_embedded.auditRecordResourceList")
-					.description("Contains a collection of Audit Records"),
-				fieldWithPath("_links.self").description("Link to the audit record resource"),
-				fieldWithPath("page").description("Pagination properties"))));
+					subsectionWithPath("_embedded.auditRecordResourceList")
```

Comment:
Was it wrong when we used `fieldWithPath` before?

Reply:
This was me getting lazy as at this point I realized that you can just use `subsectionWithPath` without defining all fields. New restdocs version forces us to choose either way.